GAN_Assignment4_Methods/src/gan_assignment4_methods/GAN_Assignment4_Methods.java

```diff
@@ -0,0 +1,180 @@
+/*
+Gary A. Newsome
+CPT 163_27 Java Programming 1
+Westmoreland County Community College
+Paul Scarrone
+ */
+package gan_assignment4_methods;
+import java.util.Scanner; // Import Scanner utility
+
+/**
+ * Convert Assignment 2 Conditional Statement Menu
+ * to a menu using methods
+ * @author Gary A. Newsome
+ */
+public class GAN_Assignment4_Methods {
+
+    /**
+     * @param args the command line arguments
+     */
+    public static void main(String[] args) {
+       // Create Scanner
+            Scanner keyboard = new Scanner(System.in);
+        // Declarations
+            int intMenuSelect; // Menu selection
+        
+        do{    
+        // Menu
+        println("Greetings, welcome to the menu\n"
+            + "Please choose from one of the options listed below:\n"
+            + "--------------------------------------------------\n"
+            + "1.) Run the default ASCII word counting program from\n"
+            + "\tweek 1. This will give you the value of the word 'Gary'\n"
+            + "2.) Run the alternate ASCII word counting program\n"
+            + "\tThis alternate will give you the value of\n"
+            + "\ta word of your choosing\n"
+            + "3.) Say 'Hi!'.\n"
+            + "4.) Super top secret.\n"
+            + "5.) Contact the programmer.\n"
+            + "\nPress 0 (zero) to exit.");
+        
```

Comment:
Very nice program.  I thought it was very well documented.  I had no problem following along.  I ran it in Netbeans and my only suggestion is that it was a little hard to follow when the full menu displayed again.  I like that it keeps running until l choose 0.  Maybe if you just gave the user the option of making another selection from the original menu?  (Note: I am just trying to provide some constructive criticism.  This is way better than mine. :smile: )